features/admin.webhooks.v1/components/webhook-channel-config-form.scss

```diff
@@ -31,7 +31,7 @@
 
         .MuiFormControlLabel-root {
             align-items: flex-start !important;
-            margin: 0;
+            margin: 0 !important;
```

Comment:
Is this getting overriden with some other style?
We may be able to get rid of the `!important`.

Reply:
Checked and removed unnecessary overrides.